### PR TITLE
Fixating the timezone of our application in PST

### PR DESCRIPTION
**Pull request by Kyubinhan**
* Add luxon(new time/date handler) to the project & remove the moment library
* Stop using Date object in graph components

Test: https://wps-pr-766.pathfinder.gov.bc.ca/morecast?codes=322

I tested this with a plugin called spoof timezone: https://chrome.google.com/webstore/detail/spoof-timezone/kcabmhnajflfolhelachlflngdbfhboe?hl=en
It randomly sets your browser timezone and you can verify by running `new Date()` in your console.
I confirmed the changes seem to be working!

### CODE REVIEW

**Review by Sybrand — APPROVED** [review]
Agree to disagree

File: web/src/features/fireWeather/components/graphs/plotlyHelper.ts [hunk 1]
```diff
@@ -95,24 +89,24 @@ export const populateGraphDataForTempAndRH = (
   tempPlumeColor?: string,
   rhPlumeColor?: string
 ) => {
```
Comment (andrea-williams) [inline web/src/features/fireWeather/components/graphs/plotlyHelper.ts:91]: I'm seeing a warning (presumably from sonarQube) about "Missing return type on function" - could you address this please?

Comment (andrea-williams) [reply web/src/features/fireWeather/components/graphs/plotlyHelper.ts:91]: It's also happening in a few other functions further down in the file

Comment (Kyubinhan) [reply web/src/features/fireWeather/components/graphs/plotlyHelper.ts]: Done!

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**